### PR TITLE
Add vertical align stepper. Fix displaying on mobile devises. Remove …

### PR DESCRIPTION
**Pull request by KirillMitskevich**
…scroll if root page

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://update-stepper.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://update-stepper.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/concepts/stepper/assets/setting.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 @mixin stepper-transition($properties...) {
     transition-timing-function: ease-in-out;
-    transition-duration: 0.2s;
+    transition-duration: 0.3s;
```
Comment (evgenTraytyak) [inline app/concepts/stepper/assets/setting.scss]: it's very slow

File: app/concepts/stepper/assets/setting.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     transition-property: $properties;
     }
 
 $stepper__item--offset: 50%;
-$stepper__item--delta: 10%;
+$stepper__item--delta: 0;
```
Comment (evgenTraytyak) [inline app/concepts/stepper/assets/setting.scss]: delta is required for animation

Comment (KirillMitskevich) [inline app/concepts/stepper/assets/setting.scss]: just don't use delta, but set another values manually

File: app/concepts/stepper/assets/setting.scss (hunk 1 above)
Comment (KirillMitskevich) [inline app/concepts/stepper/assets/setting.scss]: in works pretty nice I think. locally

**Pull request merged**